config/config.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
This file shouldn't be part of the common config either. This is a pymodbus specific and only used for testing. Therefore, please move this file to a testing folder.